go.mod

```diff
@@ -3,19 +3,27 @@ module github.com/codeready-toolchain/registration-service
 go 1.12
 
 require (
+	github.com/dgrijalva/jwt-go v3.2.0+incompatible
+	github.com/dnaeon/go-vcr v1.0.1 // indirect
 	github.com/evanphx/json-patch v4.5.0+incompatible // indirect
+	github.com/fabric8-services/fabric8-auth-client v0.0.0-20190313133658-59d93fb7492b
+	github.com/fabric8-services/fabric8-common v0.0.0-20190529115613-c08218d5adf2
```

Comment:
You are copy-pasted the fetching key thing from https://github.com/fabric8-services/fabric8-common/blob/b10e057d860d730661b2440dc1326c7d82606acc/auth/jwk/fetch_keys.go (which is fine), so, I guess we don't need any fabric8-* stuff here? Just wonder if it's some leftovers from you local experiments.

Reply:
Yeah, might be. Will fix that.

Reply:
Even then, this should be updated on a build. Wondering why this does not work...

Reply:
Fixed 31a3d4a.